app/assets/stylesheets/govuk_publishing_components/components/_print-link.scss

```diff
@@ -0,0 +1,30 @@
+.js-enabled {
+  .gem-c-print-link {
+    display: block;
+  }
+}
+
+.gem-c-print-link.gem-c-print-link__show-without-js {
+  display: block;
+}
+
+.gem-c-print-link {
+  display: none;
+  margin-top: 2em;
+  margin-bottom: 2em;
+
+  .gem-c-print-link__link {
```

Comment:
Could this class be un-nested from the `.gem-c-print-link` class? We try to avoid nesting if possible (and sorry, this probably isn't your code if you've copied it directly from government-frontend).

Reply:
Ah no worries - yeah I though `link__link` was a bit peculiar. I've un-nested and removes the extra div wrapping the link.

Reply:
Ah, sorry, I didn't quite mean that. I mean instead of 

```
.gem-c-print-link {
   .gem-c-print-link__link { }
}
```

just un-nest it like so:

```
.gem-c-print-link { }
.gem-c-print-link__link { }
```

I don't think you should remove the wrapping div - it looks like it's broken the tests and also the styling on focus:

<img width="903" alt="Screenshot 2020-06-23 at 10 08 52" src="https://user-images.githubusercontent.com/861310/85385022-b6a12980-b539-11ea-91f5-6a110da415e4.png">

Should look like this:

<img width="908" alt="Screenshot 2020-06-23 at 10 10 42" src="https://user-images.githubusercontent.com/861310/85385120-d2a4cb00-b539-11ea-8425-48b75f0b6504.png">